tmt/libraries/beakerlib.py

```diff
@@ -207,6 +208,10 @@ def fetch(self) -> None:
         # Check if the library was already fetched
         try:
             library = self._library_cache[str(self)]
+            # Check in case "tmt try retest" deleted the libs
+            assert self.parent.workdir
+            if not os.path.exists(self.parent.workdir / self.dest / self.repo):
```

Comment:
We switched from `os.path` to https://docs.python.org/3/library/pathlib.html, the correct way would be `if not (self.parent.workdir / self.dest / self.repo).exists()`